tests/unittest_brain.py

```diff
@@ -1473,7 +1473,7 @@ def test_collections_object_not_yet_subscriptable_2(self):
 
     @test_utils.require_version(minver="3.9")
     def test_collections_object_subscriptable_3(self):
-        """With python39 ByteString class of the collections module is subscritable
+        """With python39 the ByteString class of the collections module is subscritable
```

Comment:
```suggestion
        """With Python 3.9 the ByteString class of the collections module is subscritable
```
The executable is `python3.9` with the dot being significant.